int32 id1
int32 id2
string type
geometry_msgs/Pose2D measurement
float32[3] measurement_sigmas
float32[9] measurement_covariance
